### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by dju90**
Update privacy policy with recent edits from privacy counsel, including details on third party service providers.

All changes approved by Hadi.

## Links

- [spec](https://docs.google.com/document/d/1ETHmj3XACmKO3fG1wWb380_fDHzeDeX4Thq8ELe_xiA/edit#)
- [jira](https://codedotorg.atlassian.net/browse/FND-1282)

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[FND-1282]: https://codedotorg.atlassian.net/browse/FND-1282

### CODE REVIEW

File: dashboard/app/views/home/_privacy.md [hunk 1]
```diff
@@ -122,13 +122,11 @@ A pixel tag (also known as a “clear GIF” or “web beacon”) is a tiny imag
...
+Code.org may use a variety of third-party service providers to support our operations.  For example, we may use third parties such as email service platforms to send email, analytics companies to understand our Website usage and performance, and social networking platforms to host our videos. Similarly, we may use third-party service providers to implement and host our Website and associated services and features, provide teacher forums, process donations, provide contract and workflow management services, assist with customer support, and provide other functions in support of our organization.  When we give service providers access to data, including Personal Information, they are only allowed to use the data to provide services for which we’ve contracted and based on our direction. They are not allowed to use Personal Information for any other reason without the user’s consent or at the user’s direction.  You can find a list of third-party service providers we use [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ_AYYT_4K9Hpc5LrL4QAeXUmEMQR7rAXrHtloM4yt3xNndqPh-ABHXy0SJHZQ8ZSDSFQQv7ZWdXQjj/pubhtml?gid=222345124&single=true).
```
Comment (breville) [inline dashboard/app/views/home/_privacy.md]: It looks like the URL can have everything after `pubhtml` removed?

File: dashboard/app/views/home/_privacy.md [hunk 2]
```diff
@@ -275,8 +272,8 @@ If you choose to use our Website from the European Union, please note the follow
 
 ### Conditions of Use and Notices
 
-If you choose to visit our Website, your visit and any dispute over privacy is subject to this Privacy Policy and our Terms of Service, including limitations on damages, arbitration of disputes, application of the law of the State of Washington and venue for all disputes in Seattle, Washington.
-
+If you choose to visit our Website, your visit and any dispute over privacy is subject to this Privacy Policy and our Terms of Service, including limitations on damages, arbitration of disputes, application of the law of the State of Washington and venue for all disputes in Seattle, Washington.  Where we have entered into separate agreements with applicable local education authorities, such as our [Student Data Privacy Addendum](/dpa), the terms of those agreements may also govern and/or take precedence.
```
Comment (breville) [inline dashboard/app/views/home/_privacy.md]: If this is shown on dashboard, do we need the `/dpa` URL to also include the `https://code.org` domain?

Comment (dju90) [reply dashboard/app/views/home/_privacy.md]: ooh good catch

File: dashboard/app/views/home/_privacy.md (hunk 1 above)
Comment (dju90) [reply dashboard/app/views/home/_privacy.md]: thanks, done!

**Review by breville — APPROVED** [review]
Without reviewing content, the "code" surrounding it seems okay here.

File: dashboard/app/views/home/_privacy.md [hunk 3]
```diff
@@ -122,13 +122,11 @@ A pixel tag (also known as a “clear GIF” or “web beacon”) is a tiny imag
...
... content outside of our curriculum. We have also tagged our “YouTube” videos on the site for “child-directed treatment”. Schools can choose to block access to YouTube, in which case we use a fallback option that plays videos directly from our Website. We also offer clearly marked links to visit Facebook or Twitter to share various things such as your creations or certificates on Code.org. Using an account on these third party social networking services is entirely at your option and under your control, and will not result in behavioral tracking of your browsing behavior on Code.org. However, because we know that these services use behavioral targeting as part of their advertising business model, on student-facing course and activity pages, we don’t offer links to these services to Students under the age of 13, or in schools that have blocked Internet access to these services. (To opt out of various forms of internet advertising by third parties, you may visit the following links: http://www.networkadvertising.org/choices or http://www.aboutads.info/choices/. If you use these tools, you may need to opt out separately for each of your devices and for each web browser on each device.)
```
Comment (clareconstantine) [inline dashboard/app/views/home/_privacy.md]: "third party" is hyphenated elsewhere

**Review by clareconstantine — APPROVED** [review]
one small note, otherwise looks good!

File: dashboard/app/views/home/_privacy.md (hunk 3 above)
Comment (dju90) [reply dashboard/app/views/home/_privacy.md]: thanks! I've fixed this grammar everywhere where it occurs

**Pull request merged**